src/mscorlib/shared/System/Runtime/CompilerServices/ConfiguredValueTaskAwaitable.cs

```diff
@@ -5,88 +5,212 @@
 using System.Diagnostics;
 using System.Runtime.InteropServices;
 using System.Threading.Tasks;
+#if !netstandard
```

Comment:
nit: add new line between `using System.*` directives and `Internal.*` ones

Reply:
ok